libpoolprotocols/stratum/EthStratumClient.cpp

```diff
@@ -462,7 +466,11 @@ void EthStratumClient::processReponse(Json::Value& responseObject)
 			if (params.isArray())
 			{
 				string job = params.get((Json::Value::ArrayIndex)0, "").asString();
-
+				if (m_response_pending && m_onSolutionStale)
+				{
+					m_onSolutionStale();
```

Comment:
Would it be better just set ``m_stale`` to true, because the accept/reject message will still be returned.

Reply:
Miner Stale:

````
 i  19:31:45|cuda-0  |  Stale nonce 0x5c0e626546860526 submitted to eu2.ethermine.org
 i  19:31:46|stratum |  **Accepted (stale)  in 38 ms.
````

Server stale:

````
 i  19:31:45|cuda-0  |  Nonce 0x5c0e626546860526 submitted to eu2.ethermine.org
 i  19:31:46|stratum |  **Accepted (stale)  in 38 ms.
````

Reply:
So no need for the extra callback method. And the output will be unified with miner stales.

Reply:
Hmm... sounds good. I'll rework.